doc/modules/clustering.rst

```diff
@@ -140,6 +140,11 @@ model with equal covariance per component.
 :term:`inductive` clustering methods) are not designed to be applied to new,
 unseen data.
 
+.. rubric:: Examples
+
+* :ref:`sphx_glr_auto_examples_cluster_plot_inductive_clustering.py`: A practical
+  example of inductive clustering model.
```

Comment:
```suggestion
* :ref:`sphx_glr_auto_examples_cluster_plot_inductive_clustering.py`: An example
of an inductive clustering model for handling new data.
```

Reply:
ok it's fine for me @virchan to commit your suggetion, but before to do it, could you add the indentation to the second line, please?